nexus/tests/integration_tests/unauthorized_coverage.rs

```diff
@@ -0,0 +1,141 @@
+        .expect("Nexus OpenAPI spec was not valid OpenAPI");
+
+    // Take each operation that we find in the OpenAPI spec, make a regular
+    // expression that we can use to match against it (more on this below), and
+    // throw them into a BTreeMap.
+    let mut spec_operations: BTreeMap<Operation, regex::Regex> = spec
+        .operations()
+        .map(|(path, method, op)| {
+            // We're going to take URLs from our test cases and match them
+            // against operations in the API spec.  The URLs from the API spec
+            // contain variables (e.g., "/instances/{instance_name}").  Our test
+            // cases have those variables filled in already (e.g.,
+            // "/instances/my-instance").
+            //
+            // To match a URL from the test case against one from the OpenAPI
+            // spec, we're going to:
+            //
+            // - use a regular expression to replace `{varname}` in the API
+            //   spec's URL with `[^/]+` (one or more non-slash characters)
+            //
+            // - use that string as the basis for a second regex that we'll
+            //   store with the Operation.  We'll use this second regex to match
+            //   URLs to their operation.
+            //
+            // This is slow (lookups will take time linear in the total number
+            // of API endpoints) and a little cheesy, but it's expedient and
+            // robust enough for our purposes.
+            //
+            // This will fail badly if it turns out that the URL contains any
+            // characters that would be interpreted specially by the regular
+            // expression engine.  So let's check up front that those aren't
+            // present.
+            assert!(
+                path.chars().all(|c| c.is_ascii_alphanumeric()
+                    || c == '_'
+                    || c == '-'
+                    || c == '{'
+                    || c == '}'
+                    || c == '/'),
+                "unexpected character in URL: {:?}",
+                path
+            );
+            let re = regex::Regex::new("/\\{[^.}][^}]+\\}").unwrap();
+            let regex_path = re.replace_all(path, "/[^/]+");
+            let regex = regex::Regex::new(&format!("^{}$", regex_path))
+                .expect("modified URL string was not a valid regex");
+            let label = op
+                .operation_id
+                .clone()
+                .unwrap_or(String::from("unknown operation-id"));
+            (Operation { method, path, label }, regex)
+        })
+        .collect();
+
+    println!("spec operations: {:?}", spec_operations);
+
+    // Go through each of the authz test cases and match each one against an
+    // OpenAPI operation.
+    let mut unexpected_endpoints = String::from(
+        "API endpoints tested by unauthorized.rs but not found \
+        in the OpenAPI spec:\n",
+    );
+    for v in &*VERIFY_ENDPOINTS {
```

Comment:
is this the only thing we're using from `super::endpoints` or are there others?

Reply:
I think it's just that -- I'll make it a more specific import.  (When I wrote that I thought there were going to be a zillion.)